### ISSUE TITLE
Skeletons are bundled with Grav 1.6.28, which fails on PHP 8

### ISSUE THREAD

**Issue by pamtbaau** [getgrav/grav#3221]
There is a [post on the discourse](https://discourse.getgrav.org/t/still-struggling-to-get-grav-to-work-0-syntax-error-unexpected-token-match/15137) about failing skeleton installations on PHP 8. The error was related to dependency `"willdurand/negotiation": "2.x-dev"`, which is a dependency of Grav 1.6. Grav 1.7 requires '"willdurand/negotiation": "^3.0"' which has solved the issue.

After downloading a few skeletons (One Page, Landio, Twenty) they all seem to be installing Grav 1.6.28 containing the failing dependency.

Although `$ bin/gpm selfupgrade` + `$ bin/gpm update` should fix the issue, shouldn't skeletons always be bundled with the latest Grav release?

**Comment by mahagr** [comment]
It's not the only library that has issues in PHP 8, many others have issues, too, which is why we couldn't update Grav 1.6 to support PHP 8.

**Comment by pamtbaau** [comment]
I can imagine...

By the way, shouldn't skeletons be powered by Grav 1.7.latest

**Comment by rhukster** [comment]
Since we moved from Travis to Github Actions, we decided we wanted to build skeletons differently.  Rather than building the skeletons when we do a release, we want to ultimately have skeletons build themselves when there's a new release on Grav side, but for now we are looking at a manual action that we can kick off after a new release is done.

Unfortunately, this requires a fair bit of rework, and we've not had chance to circle back and do that yet.
